codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostRequestEncoder.java

```diff
@@ -92,12 +92,14 @@
         HTML5
     }
 
-    private static final Map<Pattern, String> percentEncodings = new HashMap<Pattern, String>();
+    private static final SimpleImmutableEntry[] percentEncodings;
```

Comment:
@doom369 I think you can use `Entry[] percentEncodings`. Always better to use the interface and not the actually impl when possible.